scripts/tests/test-kube-deployment.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+kind create cluster --name kind
```

Comment:
Do we want to use both `kind` and `k3d`? Can we converge on a single solution instead?

Reply:
I need to delete that file, I started working on a kind based version early on before I realised it was already using k3d